README.md

```diff
@@ -94,3 +94,93 @@ python -m unittest
 ```bash
 pylint app.py apitests models routes tests util
 ```
+## How to deploy
+This chapter describes how to get a production instance running.
+### Setup a new instance
+Clone the sources:
+```bash
+git clone https://github.com/buzzn/mybuzzn-backend.git
+```
+We recommend to setup a virtual env for the projects' requirements.
+```bash
+cd PROJECT_ROOT
+virtualenv -p `whereis python | cut -d ' ' -f3` .venv
+source .venv/bin/activate
+```
+Install the requirements
+```bash
+pip install -r "requirements.txt"
+pip install git+https://github.com/buzzn/discovergy.git
+```
+Create an endpoint in your webserver, for example for apache, create a new site:
+```
+<VirtualHost *:80>
+  ServerName your-domain.net
+  ErrorLog /path/to/error/log
+  CustomLog /path/to/access.log combined
+
+  WSGIDaemonProcess mybuzznbackend user=mybuzznbackend-user group=mybuzznbackend-group threads=5 python-path=/path/to/python:/and/path/to/packages
+  WSGIScriptAlias / /path/to/mybuzzn/projectdir/mybuzznbackend.wsgi
+  WSGIPassAuthorization On # Importent, so autho header is passed to the app
+
+    <Directory /path/to/project/>
+        WSGIProcessGroup mybuzznbackend
+        WSGIApplicationGroup %{GLOBAL}
+        Order deny,allow
+        Allow from all
+    </Directory>
+</VirtualHost>
+```
+Make sure that `WSGIPassAuthorization` is set to `On`, to have the app access to
+the auth headers.
+
+Make sure a file `setup_environment.py` exists in the project root, which sets
+env variables like this:
+```python
+import os
+
+os.environ['BUZZN_SECRET_KEY'] = '...'
+os.environ['DISCOVERGY_EMAIL'] = '...'
+os.environ['DISCOVERGY_PASSWORD'] = '...'
+os.environ['JWT_SECRET_KEY'] = '...'
+os.environ['BUZZN_SQLALCHEMY_DATABASE_URI'] = '...'
+os.environ['BUZZN_PASSWORD_SALT'] = '...'
+os.environ['BUZZN_SMTP_SERVER'] = '...'
+os.environ['BUZZN_SMTP_SERVER_PORT'] = '...'
+os.environ['BUZZN_EMAIL'] = '...'
+os.environ['BUZZN_EMAIL_PASSWORD'] = '...'
+os.environ['BUZZN_BASE_URL'] = '...'
+os.environ['BUZZN_MAILER'] = '...'
+os.environ['REDIS_HOST'] = '...'
+os.environ['REDIS_PORT'] = '...'
+os.environ['REDIS_DB'] = '...'
+```
+
+Make sure the `start_redis_task.py` is running. If you are running
+systemd, this can be done using a service:
+```ini
+[Unit]
+Description=Fill redis db with current reading values
+After=network.target redis.service
+
+[Service]
+Type=simple
+Reastart=always
+ExecStart=/path/to/python/interpreter/in/environment /path/to/start_redis_task.py
+StandardOutput=journal
+User=mybuzznbackend-user
+Group=mybuzznbackend-group
+
+[Install]
+WantedBy=multi-user.target⏎
+```
+
+### Upgrading to a new version
+To upgrade to a new version, go to the project root and run `git pull`.
+If something has changed on the models, run `source ./env/bin/activate` to
```

Comment:
"If something has changed in the database models run `source ./.venv/bin/activate`..." to be consistent with the naming from above.